scripts/prereg/load_csv_prereg.py

```diff
@@ -0,0 +1,116 @@
+""" Load pre-reg questions into schema from csv file
```

Comment:
Prior to seeing this note, I did add some comments in #4280. But because this script is intended for limited use, I agree; won't push for big changes so long as this script works as intended.